edx_reverification_block/xblock/reverification_block.py

```diff
@@ -0,0 +1,69 @@
+"""An XBlock for in-course reverification. """
+
+import pkg_resources
+from xblock.core import XBlock
+from xblock.fragment import Fragment
+
+
+def load(path):
+    """Load a resource from the package. """
+    data = pkg_resources.resource_string(__name__, path)
+    return data.decode("utf8")
+
+
+class ReverificationBlock(XBlock):
+    """An XBlock for in-course reverification. """
+
+    @property
+    def course_id(self):
+        return self._serialize_opaque_key(self.xmodule_runtime.course_id)  # pylint:disable=E1101
+
+    def student_view(self, context=None):
+        """Student view to render the re-verification link
+            This will render the url to display in lms.
+        """
+        # TODO: Need to change the hard coded url
+
+        anchor_str = "<a href='http://127.0.0.1:8003/verify_student/reverify/{course_id}/midterm/'>Reverify</a>"
```

Comment:
For testing purposes, it might be nice to define a URL in workbench that presents a fake reverification view.  (Can be just a dummy page that says, "Reverification").  We can build on that to simulate different states (like we do for the fake payment page) so that we can test the XBlock display without needing to load it into the LMS.